hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -293,6 +296,12 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--checkpoint"}, description = "Resume Delta Streamer from this checkpoint.")
     public String checkpoint = null;
 
+    @Parameter(names = {"--initial-checkpoint-provider"}, description = "Generate check point for delta streamer "
+        + "for the first run. This field will override the checkpoint of last commit using the checkpoint field. "
+        + "Use this field only when switch source, for example, from DFS source to Kafka Source. Check the class "
+        + "org.apache.hudi.utilities.checkpointing for details")
```

Comment:
`InitialCheckPointProvider` did you intend to write the name of the class here?

Reply:
Do you mean we should add `InitialCheckPointProvider` here or we should remove this description?

Reply:
Changed the description to match with `--schemaprovider-class`